GiniVision/Classes/GiniVision.swift

```diff
@@ -106,7 +120,14 @@ import UIKit
             } else {
                 viewController = AnalysisContainerViewController(document: document)
             }
-            delegate.didImport?(document)
+            
+            if let didCapture = delegate.didCapture(document:) {
+                didCapture(document)
+            } else if let didCapture = delegate.didCapture(_:) {
+                didCapture(document.data)
+            } else {
+                fatalError("GiniVisionDelegate.didCapture(document: GiniVisionDocument) should be implemented")
```

Comment:
`didCapture(document: GiniVisionDocument)` or `didCapture(_:)`?

Reply:
didCapture(document: GiniVisionDocument) since it is the new one. I tried first to call it `didCapture(_ document: GiniVisionDocument)` but Xcode saw it as an ambiguity.

Reply:
I meant that you should mention both methods in the hint message. Otherwise, it's not true. The assertion will fail only if both are not implemented.